src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -207,11 +222,23 @@ export class CalcitePickListItem {
     );
   }
 
+  renderRemoveAction() {
+    if (!this.removable) {
+      return null;
+    }
+    return (
+      <calcite-action class={CSS.remove} text="remove" onClick={this.removeClickHandler}>
+        <calcite-icon scale="s" icon="x" />
```

Comment:
Can we store the icon in an `ICONS` const in the resources file? Like `ICONS.close`